crates/bitwarden-core/src/auth/login/auth_request.rs

```diff
@@ -88,20 +88,11 @@ pub(crate) async fn complete_auth_request(
     .await?;
 
     if let IdentityTokenResponse::Authenticated(r) = response {
-        let kdf = Kdf::default();
-
         client.internal.set_tokens(
             r.access_token.clone(),
             r.refresh_token.clone(),
             r.expires_in,
         );
-        client
-            .internal
-            .set_login_method(LoginMethod::User(UserLoginMethod::Username {
-                client_id: "web".to_owned(),
-                email: auth_req.email.to_owned(),
-                kdf: kdf.clone(),
-            }));
 
         let method = match res.master_password_hash {
```

Comment:
We should probably just drop this code. Support for this auth method on the approving side has been removed 3 months ago: https://github.com/bitwarden/clients/pull/14236